### PR TITLE
W-12763312-k8troubleshooting-sync-duke

### PR DESCRIPTION
**Pull request by dukesphere**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 1]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-----
-+
-If you get this error, start Docker.
+==== Docker Daemon Error
 
+//error
+include::partial$troubleshoot.adoc[tags=troubleshoot-docker-daemon-bad-response]
 
 [[bad-request]]
-* The following error can occur when you use `docker run` to register your gateway:
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:488]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 2]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
 
 [[helm-troubleshooting]]
 === Helm Command Issues
 
-Troubleshoot errors returned by a `helm` command:
+[[troubleshoot-helm-intro]]
+//intro
+include::partial$troubleshoot.adoc[tags=troubleshoot-helm-intro]
 
-[[yaml-empty]]
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:513]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 3]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-registerSecretName, registration.content or registration.secretName is required!
-----
-+
-When this error occurs, make sure that the `registration.yaml` file is not empty. If the file is empty, reregister your gateway with a new, unique name to generate a new `registration.yaml` (see <<register-flex>>) and rerun the procedures that follow the registration step. 
+[[troubleshoot-helm-yaml-empty]]
+==== Upgrade Failed: Execution Error
+//error: 
+include::partial$troubleshoot.adoc[tags=troubleshoot-helm-yaml-empty]
 
-[[cluster_unreachable]]
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:523]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 4]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-error: the server doesn't have a resource type "services"
-----
-+
-For additional information, see <<check-cluster>>.
+[[troubleshoot-helm-upgrade-failed]]
+==== Upgrade Failed: Timed Out
+//error: 
+include::partial$troubleshoot.adoc[tags=troubleshoot-helm-upgrade-failed]
 
-[[upgrade-failed]]
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:558]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 5]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-Error: UPGRADE FAILED: timed out waiting for the condition
-----
-+
-This error typically means that your cluster configuration is incorrect or that your cluster configuration tool does not support this process. See <<kubernetes-cluster-reqs>>. One potential cause is the failure to generate a `Service` resource of the `LoadBalancer` type. You can troubleshoot further by running the following commands:
+[[troubleshoot-helm-cluster-unreachable]]
+==== Kubernetes Cluster Unreachable
+//error: 
+include::partial$troubleshoot.adoc[tags=troubleshoot-helm-cluster-unreachable]
 
-** {empty}
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:567]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 6]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-+
-[source,kubectl]
-----
-kubectl -n gateway describe deployment
-----
+See <<kubernetes-cluster-reqs>> for additional information.
 
-** {empty}
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:574]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc [hunk 7]
```diff
@@ -467,130 +467,51 @@ If you encounter an error when running a command, check for troubleshooting info
...
-[source,kubectl]
-----
-kubectl -n gateway describe service
-----
+[[troubleshoot-helm-openshift-rbac]]
+==== Failed to Install CRD
+//error: 
+include::partial$troubleshoot.adoc[tags=troubleshoot-helm-openshift-rbac]
 
-//cluster-level access when rbac enabled, as for OpenShift
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-gateway-k8-getting-started.adoc:581]: moved to troubleshoot.adoc partial

File: gateway/1.4/modules/ROOT/pages/flex-troubleshoot-docker.adoc [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+include::partial$troubleshoot.adoc[tags=troubleshoot-docker-intro]
+
+[[daemon-bad-response]]
+== Docker Is Not Started
+
+//error
+include::partial$troubleshoot.adoc[tags=troubleshoot-docker-daemon-bad-response]
+
+[[bad-request]]
+== 400 Bad Request
```
Comment (dukesphere) [inline gateway/1.4/modules/ROOT/pages/flex-troubleshoot-docker.adoc]:
```suggestion
== Reg Facade Error (400 Bad Request)
```

**Review by chris-campbell9 — APPROVED** [review]

**Pull request merged**